frontend/app/router.js

```diff
@@ -98,6 +97,7 @@ Router.map(function() {
   });
 
   this.route('four-oh-four', { path: "*path" });
+  this.route('scrollable-route');
```

Comment:
Great! However, because this is an abstract route class, we don't need to tell the router that it's a real thing